thunder/tests/test_recipes.py

```diff
@@ -11,9 +11,12 @@
 from torch.testing import assert_close
 from thunder.recipes import HFTransformers
 from thunder.executors import nvfuser_available
+from thunder.executors.cudnnex import cudnn_available
+from thunder.tests.framework import version_between, IS_WINDOWS
 from thunder.tests.framework import IS_WINDOWS
 
 
+@pytest.mark.skipif(not cudnn_available(), reason="cuDNN is not available")
```

Comment:
Is there a plan to implement a CI job without cuDNN support to test these skips?

Reply:
No. These skips are for the convenience of people not having (a compatible version of) cudnn installed  on the development machine.